src/lib/layouts/Canvas.svelte

```diff
@@ -19,6 +19,20 @@
 	/** @type {Boolean} [pointerEvents] Set this to `false` to set `pointer-events: none;` on the entire layer. */
 	export let pointerEvents = undefined;
 
+	/** @type {Object|null} [attrs] An object that sets additional attribute values onto the `<canvas>` tag*/
+		export let attrs = null;
+
+	$: if (element && attrs) {
+		const attrKeys = Object.keys(attrs);
+		for (let i = 0; i < attrKeys.length; i++) {
+			const attr = attrKeys[i];
+			const property = attrs[attr];
+			if (!element.hasAttribute(attr) || element.getAttribute(attr) !== property) {
```

Comment:
Let's say you wanted to dynamically change the label post-render, would this conditional prevent that because the `element.hasAttribute(attr)` won't set any attribute that is already set?

Reply:
Probably getting rid of `!element.hasAttribute(attr) || ` would be good.